pkg/common/utils.go

```diff
@@ -81,3 +81,22 @@ func GetAccountNumFromAWSID(AWSID string) (string, error) {
 	accountNum := (z & mask) >> 7
 	return fmt.Sprintf("%012d", accountNum), nil
 }
+
+// SliceContainsString searches a slice to determine if it contains a specified string.
+// Returns the index of the first match in the slice.
+func SliceContainsString(origTargetString string, stringSlice []string, ignoreCase bool) (bool, string, int) {
+	targetString := origTargetString
```

Comment:
question: How come you created `targetString` and `stringFromSlice` vars as opposed to updating the existing values? Was it solely for clarity?
```
if ignoreCase {
    origTargetString = strings.ToLower(origTargetString)
}
```

similarly for:
```
if ignoreCase {
    origStringFromSlice = strings.ToLower(origStringFromSlice)
}
```